dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -231,6 +235,98 @@ control dash_ingress(
         const default_action = deny;
     }
 
+    action set_meter_policy_attrs(@Sai[type="sai_ip_addr_family_t", isresourcetype="true"] bit<32> ip_addr_family) {
+        if (ip_addr_family == 0) /* SAI_IP_ADDR_FAMILY_IPV4 */ {
+            if (meta.is_overlay_ip_v6 == 1) {
+                meta.dropped = true;
+            }
+        } else {
+            if (meta.is_overlay_ip_v6 == 0) {
+                meta.dropped = true;
+            }
+        }
+    }
+
+    @name("meter_policy|dash_meter")
+    @Sai[isobject="true"]
+    table meter_policy {
```

Comment:
I did not see requirement in HLD for what this table is trying to achieve? Is it needed here as part of metering? What this table is doing mostly is dropping cross address family packets.. Vendors may be able to achieve it in a different way. Is it required as part of metering policy? If so, may be we should update requirement section in HLD

Reply:
This table corresponds to the DASH_METER_POLICY table mentioned in the HLD. It is used to create v4 and v6 meter policy table instances which are then associated with ENIs. This is similar to the ACL_GROUP table for ipv4 and ipv6 address families. It is not necessary for vendor implementations to introduce this table in their datapath as long as behavior is the same